src/patterns/data/default/entity_attribute.tsv

```diff
@@ -1964,7 +1964,7 @@ OBA:VT1000369	femur size trait	UBERON:0000981	femur	PATO:0000117	size
 OBA:VT1000414	diaphragm mass	UBERON:0001103	diaphragm	PATO:0000125	mass
 OBA:VT1000420	head mass	UBERON:0000033	head	PATO:0000125	mass
 OBA:VT1000421	small intestine length	UBERON:0002108	small intestine	PATO:0000122	length
-OBA:VT1000425	skin mass	UBERON:0000014	zone of skin	PATO:0000125	mass
+OBA:VT1000425	skin mass	UBERON:0002097	skin of body	PATO:0000125	mass
```

Comment:
I would keep the original `UBERON:0000014	zone of skin ` entity here.
See my comments here: https://github.com/obophenotype/bio-attribute-ontology/issues/41#issuecomment-1279130088